tests/unit/community_label_propagation.c

```diff
@@ -33,19 +33,24 @@ int main(void) {
     igraph_vector_int_t initial;
     igraph_vector_bool_t fixed;
     igraph_integer_t i, j;
+    igraph_lpa_variant_t variant;
 
     /* Simple triangle graph, the output should be always one community */
     igraph_small(&g, 0, IGRAPH_UNDIRECTED, 0,  1,  0,  2,  1,  2, -1);
     igraph_vector_int_init(&membership, 0);
 
-    for (j = 0; j < 100; j++) {
+    for (variant = 0; variant < 3; variant++)
```

Comment:
Can you please refactor this to use explicit enum constants? For example, store the constants in an array and loop through that array.

Also, can you keep the opening brace after `for` on the same line, to keep with the existing style?

Reply:
Yes, that's indeed cleaner.

Change the style while at it. Doesn't astyle pick up on this btw?